### ISSUE TITLE
init scripts - missing dependency for LSBInit  (debian)  [JIRA: RCS-332]

### ISSUE THREAD

**Issue by th-m-vogel** [basho/riak_cs#718]
Please add dependency to the init-scripts for riak-cs, riak-cs-control and stanchion. Without this LSBInit will try to start/stop in parallel to riak.

For riak-cs (does depend on riak):
Required-Start:    $remote_fs $syslog riak
Required-Stop:     $remote_fs $syslog riak

For rian-cs-control and stanchion (does depend on riak-cs):
Required-Start:    $remote_fs $syslog riak-cs
Required-Stop:     $remote_fs $syslog riak-cs

**Comment by kuenishi** [comment]
Riak CS and Stanchion can be configured to run in separate nodes from Riak. So unfortunately, we can't set dependency between them.

**Comment by lukebakken** [comment]
Re-opening because I think this could be solved.

**Comment by shino** [comment]
Ping.  @lukebakken

**Comment by lukebakken** [comment]
These issues should probably be closed.

**Comment by shino** [comment]
Won't fix.
